pkg/provider/azure_storageaccount.go

```diff
@@ -216,6 +217,9 @@ func (az *Cloud) EnsureStorageAccount(accountOptions *AccountOptions, genAccount
 				Tags:     tags,
 				Location: &location}
 
+			if accountOptions.EnableLargeFileShare {
+				cp.AccountPropertiesCreateParameters.LargeFileSharesState = storage.LargeFileSharesStateEnabled
```

Comment:
better add a log here, easier for troubleshooting
```
klog.V(2).Infof("azure - ....
```

Reply:
Done